sale_order_priority/models/sale.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+from odoo.addons.procurement.models import procurement
```

Comment:
@SimoRubi As you depends on procurement addon, it is better to reflect it in the manifest.